functions.php

```diff
@@ -22,6 +24,8 @@ function gesso_theme_setup() {
 		// Disable WordPress's block patterns.
 		// Comment out if you want to use them.
 		remove_theme_support( 'core-block-patterns' );
+		// Enable block template parts
+		add_theme_support( 'block-template-parts' );
```

Comment:
```suggestion
		add_theme_support( 'block-template-parts' );
		// Remove block template support.
		remove_theme_support( 'block-templates' );
```

I found this necessary so as to be able to navigate to the template parts in the UI. Otherwise out of the box you may receive a message about FSE being disabled.

Reply:
Sorry, can you clarify? You found it necessary to have the code on line 30?
`remove_theme_support( 'block-templates' );`

Reply:
Nevermind. I am not sure what I was seeing.